core/deployment/src/main/java/io/quarkus/deployment/dev/DeploymentProblem.java

```diff
@@ -0,0 +1,18 @@
+package io.quarkus.deployment.dev;
+
+public class DeploymentProblem {
```

Comment:
Is there a good reason to introduce this vs using `AtomicReference`?

Reply:
No, old habits :). I adjusted to use `AtomicReference`.